homeassistant/components/device_tracker/services.yaml

```diff
@@ -30,7 +30,7 @@ see:
         text:
     gps:
       name: GPS coordinates
-      description: GPS coordinates where device is located (latitude, longitude).
+      description: GPS coordinates where device is located [latitude, longitude].
```

Comment:
This is not an example. The parenthesis here is explaining what "located" means. The example is below and that is using square brackets.

Reply:
If we want to avoid confusion, I'd remove the parenthesis completely.

Reply:
Good suggestion. I changed the text.